src/dispatch/__tests__/DataPlaneClient.test.ts

```diff
@@ -70,8 +70,8 @@ describe('DataPlaneClient tests', () => {
         await client.sendFetch(Utils.PUT_RUM_EVENTS_REQUEST);
 
         // Assert
-        // @ts-ignore
-        const signedRequest: HttpRequest = fetchHandler.mock.calls[0][0];
+        const signedRequest: HttpRequest = (fetchHandler.mock
```

Comment:
can be:

```ts

const [[signedRequest: HttpRequest] = fetchHandler.mock.calls;

Reply:
maniator@, does this automatically index for us?

Reply:
this would get the first index value of the first index value of the array of calls.

Can test this out in dev tools by doing the following:

```js
const [ [ signedRequest ] ] = [['req', 12], [1,2,3]];
console.log(signedRequest);
```

<img width="389" alt="image" src="https://user-images.githubusercontent.com/539579/168826280-3d10e9a5-424f-4002-a0f6-df65ef9bd5d8.png">

Reply:
Never knew that!

Reply:
The more you know :-) 

Same as object destructuring here: https://github.com/aws-observability/aws-rum-web/pull/156#discussion_r873865794 but with an array :-)

Reply:
I'm not a big fan of destructuring. In many cases its additional layer of abstraction makes the code less clear (especially for those new to JS/TS). E.g., this would become:
```
const [[signedRequest]]: [[signedRequest: HttpRequest]] = fetchHandler.mock.calls as any;
```